python/nnvm/frontend/coreml.py

```diff
@@ -248,7 +248,7 @@ def ConcatLayerParams(op, insyms, symtab):
         raise NotImplementedError("Sequence Concat not supported")
     ret = insyms[0]
     for i in range(1, len(insyms)):
-        ret = _sym.concat(ret, insyms[i], dim=1)
+        ret = _sym.concatenate(ret, insyms[i], dim=1)
```

Comment:
we can do ret = _sym.concatenate(*insyms, dim=1)

Reply:
There are also some `ret`s used as parameters nearby (like [this one](https://github.com/dmlc/nnvm/blob/a96133e4074e61054d106efab2ca4ebebbde7454/python/nnvm/frontend/coreml.py#L239)). Should I remove them in this PR?

Reply:
only concatenate support multiple input value